javascript/javascript3/week3/app.js

```diff
@@ -0,0 +1,126 @@
+import env from "./env.json" assert { type: "json" };
+
+const btnTag = document.getElementById("submitBtn");
+// const saveBtnTag = document.getElementById("saveBtn");
+const urltag = document.getElementById("url");
+const message = document.getElementById("message");
+const screenshotList = document.getElementById("screenshotList");
+const screenshotTag = document.getElementsByClassName("screenshot-wrapper");
+let screenshotUrl;
+
+// function to get screenshot
+const getData = async (urltag) => {
+  const url = `https://website-screenshot6.p.rapidapi.com/screenshot?url=${urltag}&width=1000&height=800&fullscreen=false`;
```

Comment:
You can get the screensize dynamically by using the screen object so you don't have to hardcode these:
```javascript
const screenshotImg = document.createElement('img');
screenshotImg.src = screenshotUrl;
screenshotImg.alt = 'Generated Screenshot';
screenshotContainer.appendChild(screenshotImg);
```